docs/pages/contributing/documentation/style-guide.mdx

```diff
@@ -1,9 +1,34 @@
 ---
-title: Style Guide
-description: Use this reference to ensure our docs are consistent and effective
+title: Style guide
+description: Provides guidelines, documentation conventions, and recommended practices for contributing to Teleport documentation.
 ---
 
-To keep our documentation effective as Teleport develops, we need to ensure that we maintain a consistent approach to organizing information.
+Technical documentation can be a challenging business, and this challenge can mean 
+wrangling a many-headed, many-handed beast. To help us stay on course—provide our 
+North Star—this guide describes our approach to building effective documentation, 
+the conventions we follow, and the recommended practices for organizing, crafting, 
+and evolving our information architecture with a clear and consistent voice.
+
+First things first: technical documentation is separate from marketing communications. 
+Marketing communications, brand awareness, positioning, and other types of content are 
+important and provide the underpinnings of what we do. However, our focus is on providing 
+the technical content that helps people to get work done. We strive to provide the conceptual 
+and practical information that the community needs to be successful using Teleport  
+and related technologies.
+
+This guide focuses on best practices for writing technical documentation and on the style 
+conventions to use when developing documentation for Teleport products and Teleport audiences. 
+The goal of this guide is to help members of the documentation team and any one interested 
+in contributing to documentation write material that is clear, concise, and consistent.
+
+If you can't find the answer to a style, voice, or terminology question in this guide, 
+consult the following resources:
+
+- [Google developer documentation style guide](https://developers.google.com/style)
+- [Microsoft Style Guide](https://docs.microsoft.com/en-us/style-guide/welcome/)
+- [Apple Style Guide](https://help.apple.com/pdf/applestyleguide/en_US/apple-style-guide.pdf)
+- [Chicago Manual of Style](http://www.chicagomanualofstyle.org/)
+- [Merriam-Webster Dictionary](https://www.merriam-webster.com/)
```

Comment:
Are all of these style guides consistent with one another, and are they all accessible (i.e., free) for our audience of open source contributors? 

The previous version of this page (I think the guidance might still be there) told readers that if a style question isn't addressed here, to privilege internal consistency within a page. What do you think of that?